tests/ci/submit_azureml_pytest.py

```diff
@@ -37,7 +37,7 @@
 import argparse
 import logging
 
-from azureml.core.authentication import AzureCliAuthentication
+from azureml.core.authentication import AzureCliAuthentication, ServicePrincipalAuthentication
```

Comment:
SPA looks interesting.

Reply:
NOTE this is for the AML workspace and I noticed in the current pipeline there is an SPA authentication at higher level - subscription. Is this something done by you and Richin?